.github/workflows/on-pr-review.yml

```diff
@@ -274,6 +275,86 @@ jobs:
           if [[ ! -z $(helm ls -a -n reviews |grep ${POD_NAME} |grep -v "deployed ") ]]; then helm uninstall -n reviews ${POD_NAME}; fi
           helm upgrade --install ${POD_NAME} ./helm/charts/wepublish -n reviews -f values.yaml --wait
 
+  setup-frontend-matrix:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.set-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Set up matrix
+        id: set-matrix
+        run: |
+          targets=$(find apps -mindepth 2 -maxdepth 2 -type f -name .frontend_deployment -exec dirname {} \; | xargs -n 1 basename)
```

Comment:
probably better to use nx, give the project tags and then you can get all the target that way, no?

Reply:
Also i would need to install nx and node right?

Reply:
I guess yeah you would need node.


We can add tags in the projects.json and get them like: `nx show projects --projects tag:<TAG_YOU_GAVE>`